### PR TITLE
Adding uWSGIi integration

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

Adds uWSGI integration to the doc

### Motivation
PM request

### Preview link

* https://docs-staging.datadoghq.com/gus/uwsgi/integrations/uwsgi/

### CODE REVIEW

File: content/integrations/uwsgi.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+description: Collect uWSGI logs in order tracks requests per second, bytes served, request status and more.
+short_description: Collect uWSGI logs in order tracks requests per second, bytes served, request status and more.
+ddtype: check
+git_integration_title: uwsgi
+has_logo: true
+integration_title: uwsgi
+is_public: true
+kind: integration
+name: uwsgi
+public_title: Intégration Datadog-uWSGI
```
Comment (NBParis) [inline content/integrations/uwsgi.md]: Excuse your french :)

File: content/integrations/uwsgi.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+short_description: 
+supported_os:
+- linux
+- mac_os
+- windows
+---
+
+## Overview
+
+Collect uWSGI logs in order tracks requests per second, bytes served, request status (2xx, 3xx, 4xx, 5xx), service uptime, slowness, and more.
```
Comment (cswatt) [inline content/integrations/uwsgi.md]: Collect uWSGI logs in order, track requests per second, bytes served, request status (2xx, 3xx, 4xx, 5xx), service uptime, slowness, and more.

also change the `description` and `short_description` above to have grammar consistency.

File: content/integrations/uwsgi.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+### Configuration
+
+By default uWSGI server logs to stdout. Run the following command to start logging to a file or follow [uWSGI instructions to log to a file][2]:
+
+```
+uwsgi --socket :3031 --logger file:logfile=/var/log/uwsgi/uwsgi.log,maxsize=2000000
+```
+
+Create the `uwsgi.d/conf.yaml` file in the at the root of your Agent's configuration directory.
```
Comment (cswatt) [inline content/integrations/uwsgi.md]: Create the `uwsgi.d/conf.yaml` file in the root of your Agent's configuration directory.

File: content/integrations/uwsgi.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+#### Log Collection
+
+**Available for Agent version >6.0**
+
+Collecting logs is disabled by default in the Datadog Agent, enable it in your `datadog.yaml` file with:
```
Comment (cswatt) [inline content/integrations/uwsgi.md]: Collecting logs is disabled by default in the Datadog Agent. Enable it in your `datadog.yaml` file with:

File: content/integrations/uwsgi.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+logs:
+  - type: file
+    path: /var/log/uwsgi/uwsgi.log
+    service: <MY_APPLICATION>
+    source: uWSGI
+```
+
+Finally, [restart the agent][3].
+
+By default the Datadog-uWSGI integration supports the [default uWSGI log format][4] and the [apache-like format][5].
```
Comment (cswatt) [inline content/integrations/uwsgi.md]: capitalize Apache

**Pull request merged**